.golangci.yml

```diff
@@ -11,7 +11,7 @@ linters:
     - gofmt
     - goimports
     - gosimple
-    - golint
+    - revive
```

Comment:
Could we move this down to place it in alphabetical order?

Reply:
sure!